### PR TITLE
adding multi cluster test queries

### PR DESCRIPTION
**Pull request by jessegoodier**
Please test out these queries and let me know if there are any issues.

### CODE REVIEW

File: custom-prom.md [hunk 1]
```diff
@@ -146,22 +146,67 @@ Make sure that [honor_labels](https://prometheus.io/docs/prometheus/latest/confi
...
+{"status":"success","data":{"resultType":"vector","result":[]}}
+```
+
+Good:
+
+```json
+{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"node_total_hourly_cost","instance":"aks-agentpool-81479558-vmss000001","instance_type":"Standard_B4ms","job":"kubecost","node":"aks-agentpool-81479558-vmss000001","provider_id":"azure:///subscriptions/0bd50fdf-c923-4e1e-850c-196dd3dcc5d3/resourceGroups/mc_kc-demo_kc-demo-dev_eastus/providers/Microsoft.Compute/virtualMachineScaleSets/aks-agentpool-81479558-vmss/virtualMachines/1","region":"eastus"},"value":[1673020150,"0.16599565032196045"]}]}}
+```
+
+
```
Comment (linhlam-kc) [inline custom-prom.md:165]: This is great ! I have some suggestions to move this section below the `curl` command to not confusing new/in-experience users. In addition, we should add this to the `Enterprise Multi-Cluster Test` section as well so customers do not need to scroll back to see the expected result.

File: custom-prom.md [hunk 2]
```diff
@@ -146,22 +146,67 @@ Make sure that [honor_labels](https://prometheus.io/docs/prometheus/latest/confi
...
+Bad:
+
+```json
+{"status":"success","data":{"resultType":"vector","result":[]}}
+```
+
+Good:
+
+```json
+{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"node_total_hourly_cost","instance":"aks-agentpool-81479558-vmss000001","instance_type":"Standard_B4ms","job":"kubecost","node":"aks-agentpool-81479558-vmss000001","provider_id":"azure:///subscriptions/0bd50fdf-c923-4e1e-850c-196dd3dcc5d3/resourceGroups/mc_kc-demo_kc-demo-dev_eastus/providers/Microsoft.Compute/virtualMachineScaleSets/aks-agentpool-81479558-vmss/virtualMachines/1","region":"eastus"},"value":[1673020150,"0.16599565032196045"]}]}}
```
Comment (linhlam-kc) [inline custom-prom.md:162]: Should we hide the Azure subscription ID ? `0bd50fdf-c923-4e1e-850c-196dd3dcc5d3`

**Comment by jessegoodier** [comment]
@linhlam-kc good ideas. how does this look to you?

**Comment by linhlam-kc** [comment]
> 

It looks great to me ! Thank you Jesse.

**Review by linhlam-kc — APPROVED** [review]
LGTM !

**Pull request merged**